views/eApostilles/startPage.ejs

```diff
@@ -1,5 +1,5 @@
 <div class="inner_header no-user-signed-in">
-    <a href="/select-service" class="back-to-previous govuk-link">Back</a>
+    <a href="/eligibility-question-three" class="back-to-previous govuk-link">Back</a>
```

Comment:
would a user be able to go straight to this page? could you always guarantee the only way to see this content is via /eligibility-question-three?